firefox-ios/Client/Coordinators/Router/RouteBuilder.swift

```diff
@@ -126,7 +123,7 @@ final class RouteBuilder {
             TelemetryWrapper.gleanRecordEvent(category: .action, method: .open, object: .asDefaultBrowser)
             RatingPromptManager.isBrowserDefault = true
             // Use the last browsing mode the user was in
-            return .search(url: url, isPrivate: isPrivate, options: [.focusLocationField])
+            return .search(url: url, isPrivate: false, options: [.focusLocationField])
```

Comment:
do we still want this as `false`?

Reply:
I believe so, this case previously ignored any parameters in the url and relied solely on the pref for if the last session was private.